Yosemite/Yosemite/Model/Enums/CardPresentPaymentsOnboardingState.swift

```diff
@@ -101,8 +101,8 @@ extension CardPresentPaymentOnboardingState {
             return "wcpay_unsupported_version"
         case .pluginNotActivated:
             return "wcpay_not_activated"
-        case .pluginSetupNotCompleted:
-            return "wcpay_setup_not_completed"
+        case .pluginSetupNotCompleted(let plugin):
+            return plugin == .wcPay ? "wcpay_setup_not_completed" : "stripe_extension_not_setup"
```

Comment:
We have here the typical case where we would return an incorrect string if we added a new value to CardPresentPaymentsPlugin, without being notified at compile time On the other hand, we don't have any perspective of adding a new plugin in the short future, and one-liner is more straightforward than a switch. Up to you

Reply:
Yes, agree. I'll leave it as it is for the time being for simplicity and since there are no other plugins we'll be adding anytime soon 👍